project_task_code/models/project_task.py

```diff
@@ -41,3 +41,11 @@ def name_get(self):
             name = "[{}] {}".format(rec.code, task[1])
             new_result.append((rec.id, name))
         return new_result
+
+    @api.model
+    def name_search(self, name, args=None, operator="ilike", limit=100):
+        args = args or []
```

Comment:
What's the point of this line if args are just passed down?

Reply:
> I see that it's used in super, I'm just curious why you'd need to make it empty list if it's defaulted to None. I think super is capable of handling None as well, as that's the default signature of name_search, no?

Yeah, you're right, this line is pointless, I'll remove it